bluebottle/cms/serializers.py

```diff
@@ -130,7 +130,10 @@ def get_projects(self, obj):
         projects = Project.objects.filter(
             campaign_ended__gte=self.context['start_date'],
             campaign_ended__lte=self.context['end_date'],
-            status__slug__in=['done-complete', 'done-incomplete'])
+            status__slug__in=['done-complete', 'done-incomplete']
+        ).order_by(
+            'campaign_ended'
```

Comment:
@eodolphi : If we have to have the most recent projects at the top (descending order), then shouldn't then this be '-campaign_ended' ? [django docs](https://docs.djangoproject.com/en/1.10/ref/models/querysets/#order-by)